apps/examples/mediarecorder/mediarecorder_main.cpp

```diff
@@ -67,7 +67,7 @@ class MediaRecorderTest : public MediaRecorderObserverInterface, public enable_s
 					std::cout << "SELECTED APP ON" << std::endl;
 					mr.create();
 					mr.setObserver(shared_from_this());
-					mr.setDataSource(unique_ptr<FileOutputDataSource>(new FileOutputDataSource(2, 16000, PCM_FORMAT_S16_LE, "/ramfs/record")));
+					mr.setDataSource(unique_ptr<FileOutputDataSource>(new FileOutputDataSource(2, 16000, PCM_FORMAT_S16_LE, "/mnt/record")));
```

Comment:
ramfs is correct, writing on flash all the time is not good idea, also if journaling enabled, it must raise xrun.

Reply:
fixed it.